### ISSUE TITLE
Default red caption text on dark theme

### ISSUE THREAD

**Issue by adamparker** [asciidoctor/asciidoctor-intellij-plugin#841]
The default caption for dark theme is a red text on a dark background.  This is quite hard to read unless I turn up the brightness to high on my laptop.

I have searched for a quick way to fix this but apparently I need to create my own custom css file.  

Is there a way I can copy the default theme and just change this one value, or can the default caption font colour on dark theme be changed from red to an easier to read colour like yellow?

![image](https://user-images.githubusercontent.com/9744578/130626202-47db2bb8-9224-4968-bf6d-a10279e08d37.png)

**Comment by ahus1** [comment]
Thanks for pointing out that this caption is too dark. I'll change the default caption font color to a lighter red in the upcoming release. Here's a preview: 

![image](https://user-images.githubusercontent.com/3957921/131392901-fa85391d-98a2-4deb-bc5f-65949f9aef16.png)

Expect a pre-release within a week or two. I'll update this ticket once it is available.

As a workaround, you could add some in-line CSS. Please ensure to remove it once you upgrade the plugin.

```
++++
<style>
.title,.icon{
    filter: invert(90%) hue-rotate(180deg);
}
</style>
++++
```

**Comment by ahus1** [comment]
This is fixed in pre-release 0.34.1 of the plugin. Please give it a try.

The pre-release of the plugin is available from [GitHub releases](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/installing-a-preview-release.html) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).
